evap/staff/templates/staff_evaluation_textanswers_section.html

```diff
@@ -11,18 +11,24 @@
                 <table class="table table-striped{% if not forloop.last %} mb-4{% endif %}">
                     <thead>
                         <tr>
-                            <th style="width: 85%">{% trans 'Text answer' %}</th>
-                            <th style="width: 15%">{% trans 'Publish' %}</th>
+                            <th style="width: 75%">{% trans 'Text answer' %}</th>
+                            <th style="width: 15%"></th>
+                            <th style="width: 10%">{% trans 'Publish' %}</th>
                         </tr>
                     </thead>
                     <tbody>
                         {% for answer in result.answers %}
                             <tr id="{{ answer.id }}">
                                 <td class="text-answer">
-                                    <a class="textanswer-link" href="{% url 'staff:evaluation_textanswer_edit' semester.id evaluation.id answer.id %}">{{ answer.answer|linebreaksbr }}</a>
                                     {% if answer.original_answer %}
-                                        <br />
-                                        <span class="textanswer-original">({{ answer.original_answer|linebreaksbr }})</span>
+                                        ({{ answer.original_answer }}
+                                    {% else %}
+                                        {{ answer.answer }}
+                                    {% endif %}
```

Comment:
the `answer` should always be shown - the `original_answer` is then shown additionally if it exists. so the previous behavior (and styling) was correct, please only remove the `a` tag.